### DISCUSSION TITLE
add support for `all` property

### DISCUSSION THREAD

**Discussion by eliabexp** [tailwindlabs/tailwindcss#13082] [Ideas]
I think tailwind should have support for [CSS all property](https://developer.mozilla.org/en-US/docs/Web/CSS/all)

My suggestion for class names are **all-initial**, **all-unset**, **all-revert**...

**Comment by KennethHoff** [comment]
how often do you use this? I suggest creating your own variant to start with.

**Reply by eliabexp** [reply]
I think that's not only about how often someone uses it, but to make tailwind cover more functions of CSS, there is a lot of classes in tailwind that's rarely used, nonetheless, they exist to make tailwind more and more complete
